pysmiles/smiles_helper.py

```diff
@@ -295,8 +293,10 @@ def fill_valence(mol, respect_hcount=True, respect_bond_order=True,
     """
     Sets the attribute 'hcount' on all nodes in `mol` that don't have it yet.
     The value to which it is set is based on the node's 'element', and the
-    number of bonds it has. Default valences are as specified by the global
-    variable VALENCES.
+    number of bonds it has. Default valences are determined based on the charge
+    and the electron configuration from the PTE. If we can't determine the
+    valence (for example for transition metals), then the 'hcount' will be set
+    to 0.
```

Comment:
this is the default OpenSmiles treatment?

Reply:
No. OpenSmiles just enumerates some known valences for the organic subset